data/tokens.json

```diff
@@ -91,6 +91,37 @@
       }
     }
   },
+  "kovan": {
+    "baseTokenName": "ETH",
+    "genName": "GEN",
+    "tokens": {
+      "0x6f2d6ff85efca691aad23d549771160a12f0a0fc": {
+        "decimals": 18,
+        "name": "Dai Stablecoin",
+        "symbol": "DAI"
+      },
+      "0x811e5cc5fddd395d488fe92c4f0f917f3ada6ab6": {
+        "decimals": 18,
+        "name": "Sai Stablecoin v1.0",
+        "symbol": "SAI"
+      },
+      "0x6cbe14d7af0fc86c59f125c6506817d91d412582": {
+        "decimals": 2,
+        "name": "Gemini dollar",
+        "symbol": "GUSD"
+      },
+      "0x9f13dbe2ef9733a4bb580412e9cb6f463e9bdf1a": {
+        "decimals": 18,
+        "name": "KyberNetwork",
+        "symbol": "KNC"
+      },
+      "0x9be1001d601102ae0f24ab4764dd5ce2f3e5b096": {
+        "decimals": 6,
+        "name": "USD Coin",
+        "symbol": "USDC"
+      }
```

Comment:
please exclude all these tokens from kovan . 
it is not exist or the addresses are different at kovan

Reply:
done